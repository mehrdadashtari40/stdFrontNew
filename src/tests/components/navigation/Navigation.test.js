import React from 'react';
import { shallow } from 'enzyme';
import { Navigation } from '../../../common';

test('Make Snapshot file , Navigation Component', () => {
	const wrapper = shallow(<Navigation />);

	expect(wrapper).toMatchSnapshot();
});
