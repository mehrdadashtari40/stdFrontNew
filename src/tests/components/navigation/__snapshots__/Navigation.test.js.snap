// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make Snapshot file , Navigation Component 1`] = `
<aside
  id="left-panel"
>
  <Connect(LoginInfo) />
  <nav>
    <NavMenu
      closedSign="<i class=\\"fa fa-plus-square-o\\"></i>"
      openedSign="<i class=\\"fa fa-minus-square-o\\"></i>"
    />
  </nav>
  <MinifyMenu />
</aside>
`;
